rule/var_naming.go

```diff
@@ -234,6 +226,10 @@ func (w *lintNames) check(id *ast.Ident, thing string) {
 	})
 }
 
+func isUpperUnderScore(s string) bool {
```

Comment:
Please test this method with

- _
- empty string
- 404_404
- FOO_BAR
- FO
- F_O
- _FOOBAR
- FOOBAR_
- FOO123
- FOO_123

Reply:
after changes testset for ```isUpperUnderScore``` looks like this:

```
{"_", false},
{"", false},
{"empty string", false},
{"_404_404", true},
{"FOO_BAR", true},
{"FOOBAR", false},
{"FO", false},
{"F_O", false},
{"_FOOBAR", true},
{"FOOBAR_", true},
{"FOO123", false},
{"FOO_123", true},
```
of course for ```isUpperCaseConst``` some cases from above testset especially:

```
{"FOOBAR", true},
{"FO", true},
{"F_O", true},
{"FOO123", true}
```
results as true

Reply:
Thanks for adding these tests.

They are valuable for any future refactoring

Reply:
no problem :P